flink-core/src/main/java/org/apache/flink/core/fs/FileSystem.java

```diff
@@ -328,116 +360,54 @@ public static FileSystem getUnguardedFileSystem(URI uri) throws IOException {
 			final FSKey key = new FSKey(uri.getScheme(), uri.getAuthority());
```

Comment:
Stephan: Add port here as well if possible, to support multiple separate HDFS installations.

Reply:
Actually. `authority` already includes the port (authority = host + port).